### DISCUSSION TITLE
Sliders and progress bars in notification portal.

### DISCUSSION THREAD

**Discussion by RushingAlien** [flatpak/xdg-desktop-portal#1429] [Ideas]
Hello! I wonder if there are interests in having a silder/progress bar system for notifications. This would be a live notification where the app that owns it will constantly update the status. This is useful for file managers doing file operations, browsers doing downloads/uploads, torrent clients, download managers and many more.

**Comment by Mikenux** [comment]
Hello @RushingAlien!

It was considered adding progress bars to notifications. However, there are https://github.com/flatpak/xdg-desktop-portal/discussions/1236 and https://github.com/flatpak/xdg-desktop-portal/discussions/1276 (which are the same for the progress part) to consider.

Additionally, there is a question of whether it is worth having an accurate reporting of progress to allow users to see if writing activity is "suspicious" and managing writing activity.
